asab/log.py

```diff
@@ -451,8 +451,14 @@ def _on_write(self):
 			try:
 				self._socket.sendall(msg)
 			except Exception as e:
-				print("Error when writing to syslog '{}'".format(self._address), e, file=sys.stderr)
-				self._enqueue(msg)
+				# Contingency dump when the socket is not ready
+				print(msg.decode("utf-8"), file=sys.stderr)
+				print(
+					"Error when writing to syslog '{}': {}".format(self._address, e),
+					traceback.format_exc(),
+					sep="\n",
+					file=sys.stderr
+				)
```

Comment:
@ateska what should be printed to stderr in this case of socket failure?
- the log message that could not be logged to socket?
- `"Error when writing to syslog ..."` message with details?
- error traceback?